src/routes/github/webhook/webhook-receiver-post.test.ts

```diff
@@ -132,4 +132,73 @@ describe("webhook-receiver-post", () => {
 		expect(GithubWebhookMiddleware).toBeCalledWith(issueWebhookHandler);
 	});
 
+	it("should call pull handler", async () => {
+		req = {
+			headers: {
+				"x-hub-signature-256": "sha256=b3c1c0ce21666c0349b8f3010ce81299adcc9c4ee09b904f20f712292c19b792",
+				"x-github-event": "pull_request"
+			},
+			params: {
+				uuid
+			},
+			body: {
+				action: "opened"
+			}
+		};
+
+		await WebhookReceiverPost(req, res);
+		expect(GithubWebhookMiddleware).toBeCalledWith(pullRequestWebhookHandler);
```

Comment:
I think there're three layers of assertions:

1. Expect the `(GithubWebhookMiddleware).toBeCalledWith(pullRequestWebhookHandler);` <-- this is good.
2. Expect the return of `GithubWebhookMiddleware(pullRequestWebhookHandler)` is called with a correct `context`
3. Expect `pullRequestWebhookHandler` is called with correct context with extra info.

-----

You have done `1`, which is good. I am looking for `2` and `3` to improve the test. I think `3` could be hard, need more setup on the DB. But maybe try if you can achieve `2`?

Reply:
@gxueatlassian, added assertion for verifying context.